### DISCUSSION TITLE
PR "Review Requests" on logged-in home page

### DISCUSSION THREAD

**Discussion by ok200paul** [community/community#18524] [Other Feature Feedback, Questions, & Ideas] [answered]
It'd be super if we could show the PR review requests list on a little widget on the LHS of the logged-in home page. Anyone else feelin' me? Kudos the GitHub team for continuing to make such a great product & community to date!

**Comment by siminapasat** [comment, marked as answer]
@ok200paul do you have access to our Figmas and we don't know it? 😉 We're working on some updates for the Left-hand-side of the logged-in home page as we speak. Curious if you (or others) have more things they'd like to have quick access to.

**Reply by ok200paul** [reply]
Wuhu!! That's wonderful @siminapasat - issues assigned to me would be another great little list in the LHS panel, imo. Thanks so much.
